roles/yum_repo/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# defaults file for yum-repo
+repo_tag: "generic"
+repo_name: "generic"
+repo_baseurl: "generic"
```

Comment:
I think all of these above are required, so I would prefer it to fail if these are not passed in.

Therefore, please remove these from the defaults file.